### PR TITLE
Fix nexus rebuilds and simplify cli v1

### PR DESCRIPTION
**Pull request by tiagolobocastro**
fix(api/v1): rebuilds count per nexus

Control-plane worked around this for v0, we can now fix the control-plane for v1.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

refactor(io-engine-client/v1): simplify nexus create cli

Make uuid, size and children the only required positional arguments.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

File: io-engine/src/bin/io-engine-client/v1/nexus_cli.rs [hunk 1]
```diff
@@ -16,56 +16,71 @@ use uuid::Uuid;
...
         )
-        .arg(
-            Arg::with_name("nexus-info-key")
-                .required(true)
-                .help("Key used to persist the NexusInfo structure to the persistent store"),
-        )
         .arg(Arg::with_name("resv-type")
-            .required(true)
+            .required(false)
+            .default_value("Reserved")
```
Comment (abhilashshetty04) [inline io-engine/src/bin/io-engine-client/v1/nexus_cli.rs]: I think default should be "" ? In dataplane code None will be interpreted to WriteExclusiveAllRegs.

Comment (tiagolobocastro) [reply io-engine/src/bin/io-engine-client/v1/nexus_cli.rs]: Done

**Review by abhilashshetty04 — APPROVED** [review]
Lgtm

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**